### PR TITLE
Readability improvements to the stats package

### PR DESCRIPTION
**Pull request by rakyll**
DO NOT REVIEW

### CODE REVIEW

File: stats/aggregation_value.go [hunk 1]
```diff
@@ -81,9 +81,9 @@ func (a *AggregationCountValue) String() string {
 	return fmt.Sprintf("{%v}", *a)
 }
 
-// AggregationDistributionValue is the aggregated data for an
+// TestingAggregationDistributionValue is the aggregated data for an
 // AggregationDistributionFloat64  or AggregationDistributionInt64.
-type AggregationDistributionValue struct {
+type TestingAggregationDistributionValue struct {
```
Comment (bogdandrutu) [inline stats/aggregation_value.go]: Why do you change this to be Testing. Seems that this is used in the api not only for testing.

Comment (acetechnologist) [reply stats/aggregation_value.go]: Yes @bogdandrutu . These are the actual values that need to be exported. They are not for testing.

**Comment by rakyll** [comment]
Please do not review the changes labelled as "do not review".

**Comment by rakyll** [comment]
Sending these changes in smaller chunks in other PRs.

**Pull request closed without merging**